projects/lkft-android/vts-kernel-armeabi-v7a.yaml

```diff
@@ -0,0 +1,7 @@
+{% extends "testcases/android/vts.yaml" %}
+
+{% set test_timeout = 480 %}
+{% set job_name = "vts-kernel-armeabi-v7a" %}
```

Comment:
this should probably be test_name

Reply:
no, it's the job name, the test_name is hard coded in the test definitions part. like for the vts job, the test names is vts-test, which will be the test_suite name in lava result, like here: https://lkft.validation.linaro.org/results/4820252

Reply:
that should be changed.

Reply:
If you meant the test_name should be changed to job_name, then let's do that in a separate change.
If you meant the job_name here should be changed to test_name, then please give more explanation.